github_pr.py

```diff
@@ -436,7 +439,8 @@ def main():
     parser.add_argument('--mergecomment', default=":shipit:", help='string to look for when checking comments for "shipit" approval, during MERGE only')
     parser.add_argument('--condition-non-owner-merger', action='store_true', help='stops owner from being able to apply merge comment')
     parser.add_argument('--condition-approved-mergers', default=None, nargs='+', help='list of usernames of approved mergers')
-    parser.add_argument('--condition-approved-mergers-file', default='./MAINTAINERS.txt', nargs='?', help='file of usernames of approved mergers')
+    parser.add_argument('--condition-approved-mergers-file', action='store_true', help='check a file for list of usernames of approved mergers')
```

Comment:
it would be good to rename `condition-approved-mergers-file` to something like `condition-use-approved-mergers-file`. The current name make it seems like the argument is a file name (i know that the description says otherwise). Not a big deal though